src/app/pages/ArticlePage/ArticleHeadline/index.styles.ts

```diff
@@ -13,9 +13,9 @@ export default {
     css({
       height: `${pixelsToRem(44)}rem`,
     }),
-  liteCTA: ({ spacings, mq }: Theme) =>
+  canonicalToLiteSiteCTA: ({ spacings, mq }: Theme) =>
     css({
-      display: 'inline-block',
+      padding: `${pixelsToRem(13)}rem 0 ${pixelsToRem(13)}rem`,
```

Comment:
```suggestion
      padding: `${pixelsToRem(12)}rem 0 ${pixelsToRem(12)}rem`,
```
I think this was originally 12px looking at https://www.test.bbc.com/pidgin/articles/c7k3zrd3536o. Was this done intentionally? Just checking!

Reply:
Yes it's a weird one - I lost a pixel when I removed the outer div. I needed to add 26 pixels total to make the 44 pixel touch target. It was 24 previously

There is an argument to use a nicer round number and make the touch target larger - but I made the call to use 13pixels so I didn't change the current experience.

Reply:
![Screenshot 2025-04-14 at 15 59 41](https://github.com/user-attachments/assets/5eae5133-610c-467f-9a39-16096746dd21)
![Screenshot 2025-04-14 at 16 00 05](https://github.com/user-attachments/assets/44c5523e-f125-4cf2-8da1-b149e9401f69)